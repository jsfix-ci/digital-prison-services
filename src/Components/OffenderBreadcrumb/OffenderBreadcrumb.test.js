import React from 'react'
import { shallow } from 'enzyme'
import { OffenderBreadcrumb } from './OffenderBreadcrumb'

describe('Offender Breadcrumb', () => {
  it('should render correctly when values are present', () => {
    const match = { params: { offenderNo: 'AAAA111222' } }
    const wrapper = shallow(<OffenderBreadcrumb match={match} firstName="Bob" lastName="Smith" />)
    expect(wrapper.debug()).toMatchSnapshot()
  })
  it('should render correctly when values are not present', () => {
    const match = { params: { offenderNo: 'AAAA111222' } }
    const wrapper = shallow(<OffenderBreadcrumb match={match} />)
    expect(wrapper.debug()).toMatchSnapshot()
  })
})
