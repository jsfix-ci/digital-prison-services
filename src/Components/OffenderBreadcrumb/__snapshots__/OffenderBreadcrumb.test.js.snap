// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Offender Breadcrumb should render correctly when values are not present 1`] = `
"<span>
  Unknown
</span>"
`;

exports[`Offender Breadcrumb should render correctly when values are present 1`] = `
"<a href=\\"offenders/AAAA111222/quick-look\\">
  Smith, Bob
</a>"
`;
